pkg/mesh-networking/reconciliation/errhandler.go

```diff
@@ -85,14 +91,6 @@ func (e errHandler) HandleWriteError(resource ezkube.Object, err error) {
 	}
 }
 
-func (e errHandler) HandleDeleteError(resource ezkube.Object, err error) {
-	panic("unimplemented")
-}
-
-func (e errHandler) HandleListError(err error) {
-	panic("unimplemented")
-}
-
-func (e errHandler) Errors() error {
-	return nil
+func (e *errHandler) Errors() error {
+	return e.errs
```

Comment:
```suggestion
	return e.errs.ErrorOrNil()
```

We need to invoke this to safely convert to an `error` interface